lib/collection/src/operations/cluster_ops.rs

```diff
@@ -353,3 +359,28 @@ pub struct FinishResharding {}
 
 #[derive(Copy, Clone, Debug, Deserialize, Serialize, JsonSchema, Validate)]
 pub struct AbortResharding {}
+
+#[cfg(feature = "staging")]
+fn default_test_slow_down_duration_secs() -> f64 {
+    1.0
+}
+
+#[cfg(feature = "staging")]
+#[derive(Copy, Clone, Debug, Deserialize, Serialize, JsonSchema, Validate)]
+pub struct TestSlowDownOperation {
+    #[validate(nested)]
+    pub test_slow_down: TestSlowDown,
+}
+
+#[cfg(feature = "staging")]
+#[derive(Copy, Clone, Debug, Deserialize, Serialize, JsonSchema, Validate)]
+pub struct TestSlowDown {
+    /// Target peer ID to execute the sleep on.
+    /// If not specified, the operation will be executed on all peers.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub peer_id: Option<PeerId>,
+    /// Duration of the sleep in seconds (default: 1.0, max: 300.0).
+    #[serde(default = "default_test_slow_down_duration_secs")]
+    #[validate(range(min = 0.0, max = 300.0))]
+    pub duration_secs: f64,
```

Comment:
Imo it's fine to remove `_secs` as seconds are a base SI unit.

We also commonly use `timeout`, not `timeout_secs`.